### PR TITLE
Add a deploy workflow for demoee

### PR DESCRIPTION
**Pull request by knazart**
This adds a new GH workflow that updates EE core on demoee.org whenever a master branch is pushed to.

### CODE REVIEW

**Comment by knazart** [comment]
Although, now thinking about this... We want the `dev` branch there for now. On the other side I can change the branch in the scripts on the server.

File: .github/workflows/deploy-to-demoee.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Deploy to demoee.org
+
+on:
+    push:
+        branches: [master]
```
Comment (manzoorwanijk) [inline .github/workflows/deploy-to-demoee.yml:5]: To test the things, you can temporarily do this.
```suggestion
        branches: [master, deploy-to-demoee-org]
```

**Comment by knazart** [comment]
Permissions updated. Works now. 

Note that now there are two copies of EE Core on demoee. The active one is on the dev branch. Left it be in case we want to switch to the other version, that now has auto deployments through GH actions.

File: .github/workflows/deploy-to-demoee.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Deploy to demoee.org
+
+on:
+    push:
+        branches: [master, deploy-to-demoee-org]
```
Comment (manzoorwanijk) [inline .github/workflows/deploy-to-demoee.yml:5]:
```suggestion
        branches: [master]
```

**Review by manzoorwanijk — APPROVED** [review]
🚢 it

**Pull request merged**